webdriver/tests/bidi/script/call_function/remote_values.py

```diff
@@ -11,6 +11,9 @@
 @pytest.mark.parametrize("expression, expected", REMOTE_VALUES)
 async def test_remote_values(bidi_session, top_context, await_promise,
                              expression, expected):
+    if expected["type"] == "promise":
+        return
```

Comment:
Originally I was trying `pytest.skip`, but WPT does not support it.

So I am just skipping the test now.

An alternative could be to filter `REMOTE_VALUES` to remove the specific entry.

Reply:
Updated to filter instead.